packages/react/.eslintrc.json

```diff
@@ -43,6 +43,12 @@
     "react/jsx-one-expression-per-line": "off",
     "react/jsx-props-no-spreading": "off",
     "react/jsx-curly-newline": "off",
+    "react/default-props-match-prop-types": [0],
+    "react/destructuring-assignment": [0],
+    "react/function-component-definition": [0],
+    "react/jsx-no-constructed-context-values": [0],
+    "react/no-unstable-nested-components": [0],
+    "react/require-default-props": [0],
```

Comment:
// eslint-plugin-react updated destructuring-assignment
    "react/destructuring-assignment": [0],
    // eslint-plugin-react updated function-component-definition
    "react/function-component-definition": [0],
    // new in eslint-plugin-react
    "react/jsx-no-constructed-context-values": [0],
    // new in eslint-plugin-react
    "react/no-unstable-nested-components": [0],
    // eslint-plugin-react requires now props
    "react/require-default-props": [0],
    "react/default-props-match-prop-types": [0],